functions.sh

```diff
@@ -60,13 +60,17 @@ clean() {
 
 # Updates the indexes in the chroot environment.
 # Globals:
-#     None
+#     ENABLE_UNATTENDED_INSTALLATION
 # Arguments:
 #     None
 # Returns:
 #     None
 update_indexes() {
-    chroot_exec apt-get update
+    if [ ! -z ${ENABLE_UNATTENDED_INSTALLATION} ]; then
+        DEBIAN_FRONTEND=noninteractive chroot_exec apt-get update -q
+    else
+        chroot_exec apt-get update
+    fi
```

Comment:
`apt-get update` shouldn't prompt anything. However, if it does in some cases, it's an error which we have to fix.